security/firely-auth/firely-auth-management-app.rst

```diff
@@ -3,122 +3,18 @@
 
-.. _firely_auth_mgmt_exe:
+After you have installed, started and created your admin account, you can now manage users in the UI.
+After creation of a non-sso account, an email will be sent to the address of the account to activate the account. When this link would be expired, you can resend this email through the UI as well.
 
-Executable
-----------
+When you want to manage the users programmatically, you can use the API's that are specified in the swagger documentation which you can find at: https://localhost:5001/swagger/ (or replace localhost with the url of your Firely Auth installation).
+To be able to use the api, you will need a client token (obtained trough the client_credentials flow) for a client who has the ``AllowManagementApiAccess`` property set to true.
+It is recommended to use a specific client for this API access.
 
-Please contact us if you would like to receive the executable, either via mail at server@fire.ly or via the `Support Desk in Jira <https://firely.atlassian.net/servicedesk/customer/user/login?destination=portals>`_. 
-You can invoke the executable with:
-
-.. code:: Powershell
-
-    ./Firely.IdentityServer.ManagementApp.exe
-
-.. _firely_auth_mgmt_params:
-
-Commands and parameters
------------------------
-
-For brevity here is a full list of the available commands and their parameters.
-
-The parameter `--connectionString` or `-cs` is used multiple times. Use the same connectionstring as in :ref:`firely_auth_deploy_sql`.
-You can set the connectionstring in an environment variable ``FIRELY_IDENTITY_SERVER_USERS_CONNECTION_STRING`` so it doesn't need repeating.
-
-The parameter `--passwordHashIterations` or `-hi` is also used multiple times. Use the same passwordHashIterations as in :ref:`firely_auth_settings`.
-You can set the passwordHashIterations in an environment variable ``FIRELY_IDENTITY_SERVER_USERS_PASSWORD_HASH_ITERATIONS`` so it doesn't need repeating.
-
-- ``./Firely.IdentityServer.ManagementApp.exe```
-
-  - ``-?, -h, --help``: Show available commands
-  - ``users``: invoke user management
-
-    - ``list``: list all users
-      
-      ``-cs, --connectionString``
-      
-      ``-hi, --passwordHashIterations``
-
-    - ``create``: create a new user, specifying username and password
-      
-      ``-u, --username``: username
-      
-      ``-p, --password``: password - enclose in double quotes if it contains spaces.
-      
-      ``-cl`, --claim <name=value> [<name=value> ...]``: list of additional claims for this user, see 'AdditionalClaims' under :ref:`firely_auth_settings_userstore`
-      
-      ``-cs, --connectionString``
-      
-      ``-hi, --passwordHashIterations``
-
-    - ``delete``: delete a user so the user no longer has access
-      
-      ``-u, --username``: username
-      
-      ``-cs, --connectionString``
-      
-      ``-hi, --passwordHashIterations``
-
-.. _firely_auth_mgmt_password:
-
-Password requirements
```

Comment:
Do we have password requirements listed somewhere else?

Reply:
in the app settings